styleguide/components/21-atoms/inputs/input-select/_input-select.scss

```diff
@@ -1,10 +1,13 @@
 select {
```

Comment:
mimum 7 items of meer? of geld deze hier niet

Reply:
I'll ask Johan